test/testdata/lsp/completion/self_receiver.rb

```diff
@@ -0,0 +1,13 @@
+# typed: true
+
+def foo(xyz)
+  self.
```

Comment:
Do you want to put this method inside a module, so that the list of available completion items is less?